watcher/watcher.go

```diff
@@ -1041,12 +1041,12 @@ func (w *Watcher) EnsureCorrectRooms() error {
 		return err
 	}
 
-	podsToDelete := []string{}
-	concat := func(podNames []string, err error) error {
+	podsToDelete := []v1.Pod{}
+	concat := func(pods []v1.Pod, err error) error {
```

Comment:
It's better to use pointer to struct than the struct itself

Reply:
🤔 bold claim